### DISCUSSION TITLE
automerge with ignoreTests: false not working correctly

### DISCUSSION THREAD

**Discussion by dh-harald** [renovatebot/renovate#33051] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

self hosted gitea 1.22.4, self hosted renovatebot 39.60.0

### Please tell us more about your question or problem

Hi,

I'm using the scenario above, and with the following config renovatebot merged the PR, even if there was false tests
```
    {
      "enabled": true,
      "matchDatasources": ["docker"],
      "matchManagers": ["kubernetes"],
      "matchUpdateTypes": ["major", "minor", "patch", "pin", "digest"],
      "addLabels": ["tech/versions"],
      "ignoreTests": false,
      "automerge": true
    },
 ```
![image](https://github.com/user-attachments/assets/3cd8b368-53b0-4d59-8cae-67d1c91b8c7d)

### Logs (if relevant)

<details><summary>Logs</summary>

```
 INFO: Branch created (repository=k8s/manifests, branch=renovate-xxx-cert-manager-webhook-latest)
       "commitSha": "7b5301b734e106a62665ab7c4ddeccc4022851aa"
 INFO: PR created (repository=k8s/manifests, branch=renovate-xxx-cert-manager-webhook-latest)
       "pr": 3,
       "prTitle": "chore(deps): update xxx/cert-manager-webhook:latest docker digest to 704ed0a"
 INFO: Repository finished (repository=k8s/manifests)
       "cloned": true,
       "durationMs": 16158
```

</details>

**Comment by viceice** [comment]
renovate uses GitHub automerge by default, so you need to configure mandatory checks via branch restrictions or disable renovate platform automerge

**Reply by dh-harald** [reply]
It's gitea, but I think,  in this case, it doesn't matter.
I thought, (as the comment said), It need to be all tests passed to merge. Is it any extra config for that behavoiur?

**Reply by viceice** [reply, marked as answer]
gitea also has the automerge feature which merges when all required status check are passed. you need to explicit mark them as required in the branch protections, otherwise no one is required.

![image](https://github.com/user-attachments/assets/9b912620-86b1-4699-bcbc-52d29a672fe0)
Gitea only waits for these required status checks before automerge. So it's not a renovate thing.

Alternate disable platform automerge[^1] and let renovate do it (renovate will wait for all checks by default

[^1]: https://docs.renovatebot.com/configuration-options/#platformautomerge

**Reply by dh-harald** [reply]
thanks, it's clear now
